core/src/main/java/io/questdb/griffin/engine/table/BwdTableReaderPageFrameCursor.java

```diff
@@ -108,7 +108,7 @@ public SymbolTable newSymbolTable(int columnIndex) {
     }
 
     @Override
-    public @Nullable PageFrame next() {
+    public @Nullable PageFrame next(long skipTarget) {
```

Comment:
nit: can't we use `skipTarget` here for queries like `select * from x order by ts desc limit -1`?

Reply:
good spot, fixed here: https://github.com/questdb/questdb/pull/5939/commits/4091a1cba172650cb482cbd097e50a5784658b27